src/SqlUtils.ts

```diff
@@ -1,34 +1,30 @@
 import * as core from "@actions/core";
-import * as exec from "@actions/exec";
-import AzureSqlActionHelper from "./AzureSqlActionHelper";
+import * as mssql from "mssql";
 import Constants from "./Constants";
 import SqlConnectionStringBuilder from "./SqlConnectionStringBuilder";
 
 export default class SqlUtils {
-    static async detectIPAddress(serverName: string, connectionString: SqlConnectionStringBuilder): Promise<string> {
-        let sqlCmdPath = await AzureSqlActionHelper.getSqlCmdPath();
+    static async detectIPAddress(connectionString: SqlConnectionStringBuilder): Promise<string> {
+        core.debug(`Validating if client '${process.env.computername}' has access to Sql Server '${connectionString.server}'.`);
```

Comment:
does this get logged somewhere? If it is, would it be considered PII for GDPR?

Reply:
That's a good point. This is what the log looks like right now:
> ##[debug]Validating if client 'undefined' has access to Sql Server 'sqltoolsv12tests.database.windows.net'.

Looks like the `computername` is `null` on the hosted agents so maybe there's no point in keeping that. As for the server name, @dzsquared do you think that qualifies as PII?

I do want to point out this logging isn't new (it used to be on line 13), and sqlpackage also logs the server name, plus the database name as well.
> Publishing to database 'SqlActionTest-ubuntu-latest' on server 'sqltoolsv12tests.database.windows.net'.

Reply:
Do core.debug calls only output when debug logging is enabled? (https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging) This would be preferred since it minimizes exposure.

We're not collecting it even if it is output - which is good because it does fall into one of the user data categories.